content/en/blog/2023/announcing-notation-v1.0.0.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Notation v1.0.0 is available!
+author:  "Notation Release Team"
+date: 2023-08-10
+draft: false
+---
+
+The Notary Project maintainers are thrilled to announce the first stable release of Notation, including [notation v1.0.0](https://github.com/notaryproject/notation/releases/tag/v1.0.0), [notation-go v1.0.0](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0), [notation-core-go v1.0.0](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0), and [Notary Project specifications v1.0.0](https://github.com/notaryproject/specifications/releases/tag/v1.0.0) which are ready for production use!
+
+## What is Notation
+
+As containers and cloud native artifacts become the common unit of deployment, users want to know the artifacts in their environments are authentic and unmodified. The Notary Project is a set of specifications and tools intended to provide a cross-industry standards for securing software supply chains through signing and verification, signature portability, and key/certificate management. It aims to solve the core issue of trusting content within and across OCI compliant registries.
```

Comment:
Small verbiage suggestions: "As containers and cloud native artifacts become common deployment units, users want to be sure that the artifacts in their environments are authentic and unmodified. The Notary Project is a set of specifications and tools intended to provide cross-industry standards for securing software supply chains through signing and verification, signature portability, and key/certificate management. Notary aims to solve the core issue of “content trust” within and across OCI compliant registries."

Reply:
Thanks @Pabloshub . Updated. Note that we will use "Notary Project" as the project name, see https://notaryproject.dev/docs/faq/#notary-project-terms